client/sites/marketing/connections/services/facebook.js

```diff
@@ -14,35 +14,32 @@ export class Facebook extends SharingService {
 
 	didKeyringConnectionSucceed( availableExternalAccounts ) {
 		if ( availableExternalAccounts.length === 0 ) {
-			this.props.failCreateConnection( {
-				message: [
-					this.props.translate(
+			this.props.warningNotice(
+				<>
+					{ this.props.translate(
 						'The Facebook connection could not be made because this account does not have access to any Pages.',
 						{
 							context: 'Sharing: Jetpack Social connection error',
 						}
-					),
-					// Append temporary message to inform of Facebook API change
-					' ',
-					this.props.translate(
+					) }{ ' ' }
+					{ this.props.translate(
 						'Facebook supports Jetpack Social connections to Facebook Pages, but not to Facebook Profiles. ' +
 							'{{a}}Learn More about Jetpack Social for Facebook{{/a}}',
 						{
 							components: {
 								a: (
 									<a
-										href={ localizeUrl(
-											'https://wordpress.com/support/publicize/#facebook-pages'
-										) }
+										href={ localizeUrl( 'https://wordpress.com/support/post-to-facebook/' ) }
```

Comment:
Thanks for updating the link, the `#facebook-pages` anchor indeed doesn't exist any more.